src/ert/feature_toggling.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+NEW_SCHEDULER_ENABLED = "ERT_FEATURE_SCHEDULER" in os.environ
```

Comment:
This doesn't allow for explicit disabling via `ERT_FEATURE_SCHEDULER=0 ert ..`

Besides, it's not the "new" scheduler, it's the scheduler. The alternative is the job queue.

Reply:
Changed to ERT_FEATURE_SCHEDULER. You can now turn it off by doing `ERT_FEATURE_SCHEDULER=OFF ert`.

Reply:
Added some validation and a warning if the feature is turned on.